sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -570,7 +567,6 @@ case class AlterTableDropPartitionCommand(
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
     val table = catalog.getTableMetadata(tableName)
-    DDLUtils.verifyAlterTableType(catalog, table, isView = false)
```

Comment:
Can we remove `DDLUtils.verifyAlterTableType` then? Or it's still used in some places?

Reply:
It is used in:
1. AlterTableSetPropertiesCommand
```
 ALTER TABLE table1 SET TBLPROPERTIES ('key1' = 'val1', 'key2' = 'val2', ...);
 ALTER VIEW view1 SET TBLPROPERTIES ('key1' = 'val1', 'key2' = 'val2', ...);
```
2. AlterTableUnsetPropertiesCommand
```
 ALTER TABLE table1 UNSET TBLPROPERTIES [IF EXISTS] ('key1', 'key2', ...);
 ALTER VIEW view1 UNSET TBLPROPERTIES [IF EXISTS] ('key1', 'key2', ...);
```
3. AlterTableChangeColumnCommand, see the PR description about migration on new resolution framework